app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -265,7 +283,7 @@ private fun FavoriteParkingCard(
               modifier =
                   Modifier.align(Alignment.Center).padding(8.dp).testTag("ParkingItem_$index"))
 
-          IconButton(
+          androidx.compose.material3.IconButton(
```

Comment:
Why was this specific import added?

Reply:
I just swapped the imports. Previously the material 3 IconButton was imported and so when using our atom Icon Button we had to specify cyrcle.main...IconButton. I think for consistency it's better to import our own composables by default and specify "material3" when we need to use a more customizable version

Reply:
Fair enough, thanks for clarifying